index.html

```diff
@@ -4,15 +4,246 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Portifolio</title>
+    <title>Brhanu's Portifolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/e06c5cb839.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    <header>
-        logo
+    <!--Header layout Starts Here-->
+    <header class="header-container">
+        <nav class="nav-top">
+            <a href="#" class="header-logo">Welcome</a>
+            <button class="hamburger" id="menu-bar" type="button">
+               <i class="fa fa-solid fa-bars"></i>
+            </button>
+            <ul class="header-nav-desktop">
+                    <li><a href="">Portifolio</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Contact</a></li>
+            </ul>
+        </nav>
+        
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
+            </div>
+            <div class="menu-body">
+                <ul>
+                    <li><a href="#work-id">Portfolio</a></li>
+                    <li><a href="#about-id">About</a></li>
+                    <li><a href="#contact-id">Contact</a></li>
+                </ul>
+            </div>
+        </div>
     </header>
-    <section class="main-container">My head line
+    <!--Headline starts Here-->
+    <div class="wraper">
+    <div class="headline">
+        <div class="header-title">
+            <h1>Hey there. I’m Brhanu. </h1>
+            <p><span class="h-span">I’m a software developer</span></p>
+        </div>
+        <div class="heder-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-link">
+            <ul class="medias">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa fa-github"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <div class="head-btn">
+            <button type="button">
+                <i class="fa fa-solid fa-angle-down"></i>
+            </button>
+        </div>
+    </div>
+<!--Header Ends Here-->
+<!--Work card content starts Here-->
+    <section class="work-card" id="work-id">
```

Comment:
- [ ] This distance between your toolbar/header and headline section is way too much. Kindly fix it.
![image](https://user-images.githubusercontent.com/84156714/212355473-29426127-8a55-498f-88ff-edbebc8ff10b.png)

- [ ] The image below is your design not yet aligned to the center as mentioned by the other reviewers. Kindly find ways to fix this.
![image](https://user-images.githubusercontent.com/84156714/212356169-5789136d-1d52-4529-b04d-74b1514b73a9.png)